use/startMicrosatellite.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Calculating Basic Population Genetic Statistics from microsatellite Data"
+---
+
+# Introduction
+
+In this vignette, you will calculate basic population genetic statistics from microsatellite data using R packages. 
+
+1. Genetic diversity,
+2. Test Hardy Weinberg 
+3. $F_{is}$ and global $F_{st}$
+
+# Assumptions
+
+H0: we assume that the microsatellites are in Hardy-Weinberg equilibirum
+H1: they are not.
+
+
+
+# Data
+
+The dataset used for these analyses are a R dataset:nancycats
+It contains microsatellites genotypes of 237 cats from 17 colonies of Nancy (France). Each individuals of the 17 colonies are genotyped for 9 microsatellites.  
+
+nancycats is a genind, i.e. an object of the library adegenet
+
+# Resources/Packages required
+
+
+```{r, packages, message=FALSE}
+ 
+library(adegenet)
+library(pegas)
+```
+
+ 
+ 
+
+## Section 1: Load the data
+After loading the 'genind' object, we convert it into a `loci` objects (package pegas)
+
+```{r, load_data_show, eval=FALSE}
+nancycats  
+summary(nancycats)
+
+```
+
+
+```{r, data_conversion, eval=FALSE}
+Mydata <- genind2loci(nancycats)
+
```

Comment:
Note that this will not get execute if you set `eval=FALSE`. Hence, `Mydata` will not be defined after this chunk as long as you don't set `eval=TRUE` (or remove the `eval=` option, since `TRUE` is the default value for it).